app/views/foreman/job_templates/bootc_action.erb

```diff
@@ -0,0 +1,24 @@
+<%#
+kind: job_template
+name: Bootc Action - Katello Script Default
+model: JobTemplate
+job_category: Katello
+description_format: 'bootc %{action} %{options}'
+provider_type: script
+template_inputs:
```

Comment:
A sort of common thing is to include an option to reboot the machine once the action is done, maybe that could be used here as well https://github.com/theforeman/foreman_leapp/blob/master/app/views/foreman_leapp/job_templates/leapp_upgrade.erb#L16 ?

Reply:
Thanks for your review and the comment @adamruzicka.

From [bootc document](https://containers.github.io/bootc/man/bootc-upgrade.html):
```
However, in the future this is likely to change such that reboots outside of a `bootc upgrade --apply` do *not* automatically apply the update in addition.
```
With that information, maybe we just allow users decide if a reboot is necessary with `--apply`. option  What is your opinion, @ianballou?

Reply:
@lfu I'm asking around about if there is more information about that change. I do like the reboot strategy that Adam mentioned, it seems pretty straight-forward. I'll update here when I hear more from the bootc team.

Reply:
The future of this command is still a bit unclear, so I think we can leave the scripts as-is and update them as bootc changes in the future.